### ISSUE TITLE
Question on training process on CityScape dataset

### ISSUE THREAD

**Issue by Hustchenqi** [itijyou/ademxapp#28]
As described in [article](https://arxiv.org/pdf/1611.10080.pdf), the author said "we first resize  an image by a ratio randomly sampled from [0.7, 1.3], and then generate a sample by cropping one 500×500 subwindow at a randomly selected location." If the input image is scaled, the ground truth should also be resized. Whether it is reasonable to resize a label map? I know the authors gave different interpolation methods in "util.py"  , but I don't know if it is reasonable.

**Comment by ChidanandKumarKS** [comment]
hai all, what is the command to execute training procedure for cityscape semantic segmentation?         any  updates on training codes for semantic segmentation?
